Packet++/src/TcpReassembly.cpp

```diff
@@ -312,6 +327,7 @@ void TcpReassembly::reassemblePacket(Packet& tcpData)
 		LOG_DEBUG("Got FIN or RST packet without data on side %d", sideIndex);
 
 		handleFinOrRst(tcpReassemblyData, sideIndex, flowKey);
+		status = Success_FIN_RSTWithNoData;

```

Comment:
I'm not sure we need the word "Success" here. The status can simply be: `FIN_RSTWithNoData`